code/modules/food_and_drinks/food.dm

```diff
@@ -49,4 +49,9 @@
 					H.adjust_disgust(25 + 30 * fraction)
 			if((foodtype & BREAKFAST) && world.time - SSticker.round_start_time < STOP_SERVING_BREAKFAST)
 				SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "breakfast", /datum/mood_event/breakfast)
+			if(locate(/obj/structure/table) in range(1, get_turf(H)))
+				to_chat(H,"<span class='notice'>I love tables!</span>")
```

Comment:
I don't think having another to_chat every time you eat near a table is entirely needed.